packages/ui/src/components/w3m-zorb-ens-image/index.ts

```diff
@@ -0,0 +1,62 @@
+import { AccountCtrl, ClientCtrl } from '@web3modal/core'
+import type { FetchEnsAvatarOpts } from '@web3modal/ethereum'
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { useScript, ZorbPackageScript } from '../../utils/UiHelpers'
+import styles from './styles'
+
+@customElement('w3m-zorb-ens-image')
+export class W3mZorbImage extends LitElement {
+  public static styles = [styles]
+
+  // -- lifecycle ---------------------------------------------------- //
+  public constructor() {
+    super()
+    this.getENSAvatar()
+    useScript(ZorbPackageScript)
+  }
+
+  // -- state & properties ------------------------------------------- //
+  @property() public address = ''
+  @property() public size = ''
+  @property() public ens = ''
+
+  // -- private ----------------------------------------------------- //
+  private async getENSAvatar() {
+    try {
+      const opts: FetchEnsAvatarOpts = {
```

Comment:
You shouldn't really need to specify `: FetchEnsAvatarOpts` here, as we try to keep everything to typescripts auto-inheritance i.e. if it is wrong it should be highlighted on `.fetchEnsAvatar(opts)` level